Structure_Engine/Query/Geometry.cs

```diff
@@ -37,34 +39,49 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Gets the geometry of a Node as a point. Method required for automatic display in UI packages.")]
+        [Input("node", "Node to get the point geometry from")]
+        [Output("point", "The geometry of the node as its position")]
         public static Point Geometry(this Node node)
         {
-            return node.Position();
+            return node.Position;
         }
 
         /***************************************************/
 
+        [Description("Gets the geometry of a Bar as its CentreLine. Method required for automatic display in UI packages.")]
+        [Input("bar", "Bar to get the centre line geometry from")]
+        [Output("line", "The geometry of the Bar as its CentreLine")]
         public static Line Geometry(this Bar bar)
         {
             return bar.Centreline();
         }
 
         /***************************************************/
 
+        [Description("Gets the geometry of a Edge as its curve. Method required for automatic display in UI packages.")]
+        [Input("edge", "Edge to get the curve geometry from")]
+        [Output("curve", "The geometry of the Edge as its Curve")]
         public static ICurve Geometry(this Edge edge)
         {
             return edge.Curve;
         }
 
         /***************************************************/
 
+        [Description("Gets the geometry of a Structural surface as its geometrical 'centre plane' location surface. Method required for automatic display in UI packages.")]
```

Comment:
Gets the geometry of a structural Surface at it's centre. Method required for automatic display in UI packages.